src/app/public/modules/data-manager/data-manager-column-picker/data-manager-column-picker-context.ts

```diff
@@ -2,7 +2,13 @@ import {
   SkyDataManagerColumnPickerOption
 } from '../models/data-manager-column-picker-option';
 
+import {
+  SkyDataManagerColumnPickerSortStrategy
+} from '../models/data-manager-column-picker-sort-strategy';
+
 export class SkyDataManagerColumnPickerContext {
```

Comment:
It would be good to add specific arguments to the class constructor so that every time it gets created, it's given exactly what it needs. Today, it's possible to create a column picker context without any column options or column IDs. For example:
```
const context = new SkyDataManagerColumnPickerContext(); // Just an empty object
```
Instead, you could do:
```
export class SkyDataManagerColumnPickerContext {

  public columnPickerSortStrategy = SkyDataManagerColumnPickerSortStrategy.SelectedThenAlphabetical;

  constructor(
    public columnOptions: SkyDataManagerColumnPickerOption[],
    public displayedColumnIds: string[]
  ) { }

}
```
This way, every time the context is created it includes exactly what it needs and there's no "guesswork" on the maintainer's end.

Reply:
At this point this would be a breaking change, unless we also include a constructor with no arguments

Reply:
Is `SkyDataManagerColumnPickerContext` exposed to the consumer?

Reply:
ah no, it isn't. I'll change that.